test/intl402/Intl/getCanonicalLocales/name.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2016 Mozilla Corporation. All rights reserved.
+// This code is governed by the license found in the LICENSE file.
+
+/*---
+esid: sec-intl.getcanonicallocales
+description: Intl.getCanonicalLocales.name value and descriptor. 
+info: |
+  8.2.1 Intl.getCanonicalLocales (locales)
+  1. Let ll be ? CanonicalizeLocaleList(locales).
+  2. Return CreateArrayFromList(ll).
+includes: [propertyHelper.js]
+---*/
+
+assert.sameValue(
+  typeof Intl.getCanonicalLocales.name, 'getCanonicalLocales',
+  'The value of `Intl.getCanonicalLocales.name` is `"getCanonicalLocales"`'
+);
+
+verifyNotEnumerable(Intl.getCanonicalLocales, 'name');
+verifyWritable(Intl.getCanonicalLocales, 'name');
```

Comment:
From [ECMA262 section 17](https://tc39.github.io/ecma262/#sec-ecmascript-standard-built-in-objects):

> Unless otherwise specified, the name property of a built-in Function object, if it exists, has the attributes { [[Writable]]: false, [[Enumerable]]: false, [[Configurable]]: true }.